teuthology/__init__.py

```diff
@@ -14,10 +14,22 @@
 
 import logging
 import os
+import subprocess
 
+mydir = os.path.dirname(os.path.realpath(__file__))
```

Comment:
no good reason; tiny bit of documentation as to why we're calculating it, less chance of line overrun.  I can move it.